sdk/include/psdk/shobjidl.idl

```diff
@@ -4625,3 +4636,106 @@ interface IFolderBandPriv : IUnknown
 
     HRESULT SetNoText([in] BOOL fNoText);
 }
+
+
+#ifdef __REACTOS__
+/*****************************************************************************
+ * IQueryContinue interface
+ */
+
+[
+    uuid(7307055c-b24a-486b-9f25-163e597a28a9),
+    object,
+    pointer_default(unique)
+]
+interface IQueryContinue : IUnknown
+{
+    HRESULT QueryContinue();    // S_OK -> Continue, S_FALSE -> Cancel
```

Comment:
I don't think those comments will end up in the generated header?

Reply:
I don't think so, but this was not the purpose of having them in the .h anyways :)